csrc/scheduler/pointwise.cpp

```diff
@@ -477,7 +477,12 @@ std::unique_ptr<PointwiseParams> getPointwiseHeuristics(
   params->vectorization_factor = std::min(
       max_vect_factor,
       vectorize_helper::getVectorizationFactor(
-          runtime_info, largest_out, data_cache, break_point, reorder_map));
+          runtime_info,
+          largest_out,
+          data_cache,
+          break_point,
+          /*max_vectorization_size_in_bit=*/128,
```

Comment:
Why not using `SchedulerRuntimeInfo::getMaxVectorizationSizeInBit()`?

Reply:
To avoid suddenly changing all the scheduler heuristics. Will enable this 256-bit feature one-by-one after heuristic and performance checks.